db/migrate/20170512162050_create_terms_and_conditions.rb

```diff
@@ -0,0 +1,18 @@
+class CreateTermsAndConditions < ActiveRecord::Migration
+  def change
+    create_table :terms_and_conditions do |t|
+      t.string :name
+      t.string :title
+      t.text :text
+      t.string :version
+      t.boolean :latest, default: false
+      t.timestamps
+    end
+
+    create_table :terms_and_conditions_acceptances do |t|
+      t.string :user_uuid
+      t.references :terms_and_conditions
```

Comment:
this presumably automaticlaly indexed as foreign key id. But wonder if we should also index user_uuid?